qiskit/compiler/assemble.py

```diff
@@ -103,9 +105,12 @@ def assemble(experiments: Union[QuantumCircuit, List[QuantumCircuit], Schedule,
                 * ``avg`` returns average measurement output (averaged over number of shots).
         meas_map: List of lists, containing qubits that must be measured together.
         memory_slot_size: Size of each memory slot if the output is Level 0.
-        rep_time: Repetition time of the experiment in s.
-            The delay between experiments will be ``rep_time``.
-            Must be from the list provided by the device.
+        rep_time: Time per program execution in sec. Must be from the list provided
+            by the backend.
+        rep_delay: Delay between programs in sec. Only supported on certain
+            backends (``backend.configuration()['dynamic_reprate_enabled']`` ).
+            If supported, it will override ``rep_time``. Must be from the list
+            provided by the backends.
```

Comment:
Should share here what list you mean

Reply:
Addressed in https://github.com/Qiskit/qiskit-terra/pull/4539/commits/d192e28937f3e8826260746c7d083e272ffa2075.